### ISSUE TITLE
test set

### ISSUE THREAD

**Issue by nczub** [zhuyitan/IGTD#13]
Hello,
I'm trying to use your code for prediction based on chemistry descriptors. How can I create such images for test set, when I've already obtained model based on training set presented with IGTD? In different words, after creation of images are there any settings which I should give to test set?
